src/odemis/driver/actuator.py

```diff
@@ -2098,8 +2102,8 @@ def _doMoveAbs(self, pos):
         final_pos = cur_pos + move
         pass_ref = (cur_pos // self._cycle) != (final_pos // self._cycle)
 
-        # do referencing after i=5 moves or when passing the reference switch anyways
-        if pass_ref or self._move_num_total >= 5:
+        # do referencing after N moves or when passing the reference switch anyways
```

Comment:
Add in the comment `... or when passing the reference switch anyways (if referencing is not disabled)` or similar.